V2-Upgrade-Guide.md

```diff
@@ -369,15 +369,17 @@ const filter = buildAzureContentFilter({
 **v2:**
 ```typescript
 // Use this function instead
-const filter = buildAzureContentSafetyFilter({
+const filter = buildAzureContentSafetyFilter('input', { // For output filter, use type 'output'
   hate: 'ALLOW_SAFE',
   violence: 'ALLOW_SAFE_LOW_MEDIUM'
 });
 ```
 
 #### Azure Content Filter Changes
 
-The Azure content filter property names have been updated to use lowercase with underscores.
+The `buildAzureContentSafetyFilter()` function now requires a `type` parameter as the first argument to distinguish between input and output filter configurations.
+This change allows input and output filters to have different configuration options, such as the `prompt_shield` property that is only available for input filters.
+Additionally, the Azure content filter property names have been updated to use lowercase with underscores.
```

Comment:
[q] But why? Isn't the types based on interfaces we define? Can't we still have camel case? 
Or is it because snake case is being used in many other places and this is easier?

Reply:
It's just because it is easier to map. `buildAzureContentSafetyFilter()` implementation dynamically uses keys of the object and map the value to it. We can also hard code four lines for this and map the value.. But..